modules/client/src/client-common/constants.ts

```diff
@@ -21,6 +21,58 @@ export const UNAVAILABLE_PROPOSAL_METADATA: ProposalMetadata = {
   resources: [],
 };
 
+// TODO change rpcs to aragon ones
+export const WEB3_NODES: { [K in SupportedNetworks]: string[] } = {
+  homestead: ["https://mainnet.eth.aragon.network"],
+  goerli: ["https://endpoints.omniatech.io/v1/eth/goerli/public"],
+  matic: ["https://polygon.llamarpc.com"],
+  maticmum: ["https://endpoints.omniatech.io/v1/matic/mumbai/public"],
+};
+
+const getGraphqlNode = (netowrk: string): string => {
```

Comment:
nope, because this is called only by the the `GRAPHQL_NODES` constants, and the networks in satsuma have diffeerent names (osx-mainnet, osx-polygon...) so I'd leave it like this or create a new enum with the satsuma network names, because they are not the same as the `SupportedNetworks`

Reply:
Yes, we can do it, but i dont see the point, I mean, this is a private helper function to avoid repeating the same thing over and over

Reply:
Such a function being internal doesn't prevent from using clean code practices :)